include/groonga/util.h

```diff
@@ -23,6 +23,29 @@
 extern "C" {
 #endif
 
+/**
+ * \brief Inspect specified object and produce a textual representation.
+ *
+ * \since 4.0.8
+ *
+ * \note If \p obj is a \ref GRN_TABLE_PAT_KEY table, all keys are shown. To
+ *       limit output on \ref GRN_TABLE_PAT_KEY tables, use
+ *       \ref grn_inspect_limited instead.
+ *
+ * For example usage:
+ * ```c
+ *  grn_obj inspected;
+ *  GRN_TEXT_INIT(&inspected, 0);
+ *  grn_inspect(ctx, &inspected, obj);
+ *  printf("%.*s\n", (int)GRN_TEXT_LEN(&inspected), GRN_TEXT_VALUE(&inspected));
```

Comment:
```suggestion
 *  printf("%.*s\n", (int)GRN_TEXT_LEN(&inspected), GRN_TEXT_VALUE(&inspected));
 *  GRN_OBJ_FIN(ctx, &inspected);
```

We need `GRN_OBJ_FIN()`, could you add it?

Reply:
fix: [ceca49a](https://github.com/groonga/groonga/pull/2328/commits/ceca49a103daa1092035af04070ce706bf68dddc) Thanks!